src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -105,7 +106,7 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * Executes task in given context.
     */
   override def execute(): Unit = {
-    notifyToSubscribers(executeTask)
+    notifyToSubscribers(executeTask())
```

Comment:
That is a standard scala convention. side effecting functions should always have a `()`